osu.Game/Screens/SelectV2/PanelGroup.cs

```diff
@@ -145,5 +146,14 @@ protected override void PrepareForUse()
             titleText.Text = group.Title;
             countText.Text = Item.NestedItemCount.ToString("N0");
         }
+
+        protected override void Update()
+        {
+            base.Update();
+
+            const float pill_right_margin = 10f;
+
+            countPill.X = -pill_right_margin - TopLevelContent.X;
+        }
```

Comment:
my brain turned off.

is there a reason the count isn't added inside `TopLevelContent`?

if this is to remain, please fix the really awkward math or explain what it's doing. i have no idea.

Reply:
where does the 10f margin come from? can this be added to the 20f margin already on the drawable and be removed here?